benchmarks/movielens.ipynb

```diff
@@ -0,0 +1,942 @@
+       "      <td>10</td>\n",
+       "      <td>0:00:03.275541</td>\n",
+       "      <td>0:00:00.013025</td>\n",
+       "      <td>0.862083</td>\n",
+       "      <td>0.680735</td>\n",
+       "      <td>0.409749</td>\n",
+       "      <td>0.404197</td>\n",
+       "      <td>0:00:00.052090</td>\n",
+       "      <td>0.002158</td>\n",
+       "      <td>0.025905</td>\n",
+       "      <td>0.032218</td>\n",
+       "      <td>0.010285</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>7</th>\n",
+       "      <td>1m</td>\n",
+       "      <td>svd</td>\n",
+       "      <td>10</td>\n",
+       "      <td>0:01:37.171597</td>\n",
+       "      <td>0:00:25.726064</td>\n",
+       "      <td>0.891404</td>\n",
+       "      <td>0.698924</td>\n",
+       "      <td>0.362979</td>\n",
+       "      <td>0.362985</td>\n",
+       "      <td>0:03:16.591268</td>\n",
+       "      <td>0.010471</td>\n",
+       "      <td>0.096016</td>\n",
+       "      <td>0.087908</td>\n",
+       "      <td>0.024581</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>8</th>\n",
+       "      <td>1m</td>\n",
+       "      <td>sar</td>\n",
+       "      <td>10</td>\n",
+       "      <td>0:00:02.145341</td>\n",
+       "      <td>None</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>0:00:02.432556</td>\n",
+       "      <td>0.002482</td>\n",
+       "      <td>0.034275</td>\n",
+       "      <td>0.042438</td>\n",
+       "      <td>0.015325</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>9</th>\n",
+       "      <td>1m</td>\n",
+       "      <td>ncf</td>\n",
+       "      <td>10</td>\n",
+       "      <td>0:04:03.736511</td>\n",
+       "      <td>None</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>0:00:38.344802</td>\n",
+       "      <td>0.053582</td>\n",
+       "      <td>0.316194</td>\n",
+       "      <td>0.289283</td>\n",
+       "      <td>0.093617</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>10</th>\n",
+       "      <td>1m</td>\n",
+       "      <td>fastai</td>\n",
+       "      <td>10</td>\n",
+       "      <td>0:03:21.684542</td>\n",
+       "      <td>0:00:00.421794</td>\n",
+       "      <td>0.881981</td>\n",
+       "      <td>0.701716</td>\n",
+       "      <td>0.376375</td>\n",
+       "      <td>0.378968</td>\n",
+       "      <td>0:00:52.389096</td>\n",
+       "      <td>0.024375</td>\n",
+       "      <td>0.175191</td>\n",
+       "      <td>0.160295</td>\n",
+       "      <td>0.053431</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "    Data    Algo   K      Train time     Rating time      RMSE       MAE  \\\n",
```

Comment:
Maybe better to rename "Rating time" and "Ranking time" to "Predicting time" and "Recommending time", respectively?

Reply:
added in the next commit